### ISSUE TITLE
Rescue if fail to read biblio cache

### ISSUE THREAD

**Issue by opoudjis** [metanorma/metanorma-ietf#40]
````
   File.open(bibliocache_name, "r") do |f|
            biblio = JSON.parse(f.read)
          end
````

The JSON.parse needs to be rescued.

**Comment by ronaldtse** [comment]
The reason this fails is that:

1. The file is empty.
2. The file is corrupt (broken JSON)

Also, the line:
```
        bibliocache_name = "#{Dir.home}/.asciidoc-rfc-biblio-cache.json"
```

Should be changed to
```
        bibliocache_name = "#{Dir.home}/.metanorma-ietf-biblio-cache.json"
```

**Comment by opoudjis** [comment]
Changing the cache name will break software in production, and I find a concern with branding in an invisible config file to be silly.

OTOH, if this disrupts anyone's workflow, I'll happily blame you. :-)

Updating metanorma.com to reflect this. asciidoctor-rfc is frozen, and thus shall not be updated.

**Comment by opoudjis** [comment]
Also doing some fixes to align gem with asciidoctor 2.0.x

**Comment by ronaldtse** [comment]
> Changing the cache name will disrupt software in production (existing cached references will disappear), and I find a concern with branding in an invisible config file to be silly.

Disagree. This is a bug for incomplete migration of branding. Anyway it's fixed now, so it's OK.
